packages/homepage/content/changelog/4-0-1/index.md

```diff
@@ -0,0 +1,23 @@
+---
+banner: ./4-0-1.jpg
+title: v4.0.1
+slug: Custom Artwork
+date: 2021-02-23
+description: Add custom thumbnails to your sandboxes
+---
+
+#### Features
+
+1. Anti Phishing banner 
+2. Keyboard traversing in file tree [@jyash97](https://github.com/codesandbox/codesandbox-client/pull/5505)
+3. Logging for errors when installing private package 
+4. New Over-mind typing 
+5. Custom Cover Images for sandboxes 
+
+
+
+#### Fixes
+- Export ordering when re-exporting an import 
+- Yarn lockfile was updated [@NullVoxPopuli](https://github.com/codesandbox/codesandbox-client/pull/5501)
```

Comment:
This is more of an internal maintenance, so I would not put this one in explicitly.

```suggestion
```